internal/pkg/cli/svc_package.go

```diff
@@ -356,7 +346,10 @@ func (o *packageSvcOpts) getSvcTemplates(env *config.Environment) (*wkldCfnTempl
 	if err != nil {
 		return nil, fmt.Errorf("generate workload %s template against environment %s: %w", o.name, o.envName, err)
 	}
-	return &wkldCfnTemplates{stack: output.Template, configuration: output.Parameters}, nil
+	return &wkldCfnTemplates{
+			template:   output.Template,
+			parameters: output.Parameters},
+		nil
```

Comment:
nit: wut is this `gofmt` valid?

Reply:
lol yes, but i'll move it up to the other line haha